frontend/src/AppBuilder/RightSideBar/PageSettingsTab/PageMenu/PagesSidebarNavigation.jsx

```diff
@@ -69,6 +73,9 @@ export const PagesSidebarNavigation = ({
   const [measuredHeaderWidth, setMeasuredHeaderWidth] = useState(0);
   const [measuredDarkModeToggleWidth, setMeasuredDarkModeToggleWidth] = useState(0);
   const [measuredMoreButtonWidth, setMeasuredMoreButtonWidth] = useState(0);
+  const [isHovered, setIsHovered] = useState(false);
```

Comment:
@vjaris42 Can we move this to CSS? This causes unwanted re-renders on the component